betterrolls-swade2/scripts/settings_config.js

```diff
@@ -64,7 +64,11 @@ export class SettingsConfig extends FormApplication {
     s.id = s.key;
     s.name = game.i18n.localize(s.name);
     s.hint = game.i18n.localize(s.hint);
-    s.value = s.value !== undefined ? s.value : game.i18n.localize(s.default);
+    s.value = s.value !== undefined ? s.value : 
```

Comment:
This can be simplified to a null coalescing operator (yes, I know it wasn't that way before).
`s.value = s.value ?? s.default;`

Reply:
simplified. Site suggested I add some conditions to s.default to keep it consistent for localization so those have been added.

Reply:
Yeah, sometimes the AI is wrong.